Meetings/2022-03-09-meeting-minutes.md

```diff
@@ -0,0 +1,73 @@
+# Maintainers' Meeting 2022-03-09
+
+## Meeting Attendees
+
+- Rob Prouse - @rprouse
+- Mattias Karlsson - @devlead
+- Daniel Valadas - @valadas
+- Dylan Perks - @Perksey
+- Glenn Watson - @glennawatson
+- Halil İbrahim Kalkan @hikalkan
+- James Turner - @Turnerj
+- Michael Hawker - @michael-hawker
+- Nils Andresen - @nils-a
+- Pascal Berger - @pascalberger
+- Robin Krom - @Lakritzator
+
+## Recording
+
+[2022-03-09 Meeting Recording](https://dotnetfoundation-my.sharepoint.com/:v:/g/personal/rob_prouse_dotnetfoundation_org/EYdWzSDu0qVPkN8vPRw6Ux8BuF8SMfimpSpWlCwCVxa3QA?e=KFvGBC)
+
+### Virtual Inc Staff Observers
+
+- Tom Pappas
+- Nicole Miller
+
+## Slides
+
+## Agenda
+
+- Review previous [meeting minutes](https://github.com/dotnet-foundation/wg-maintainers/blob/main/Meetings/2022-02-16-meeting-minutes.md)
+- Briefly discuss ()[https://github.com/dotnet-foundation/wg-maintainers/issues/48]
+- Talk about how we can provide better guidance to maintainers from this thread as well: [Centralised information for .NET Foundation operation](https://github.com/dotnet-foundation/Home/discussions/61)
+- Discuss #55 and #56
+- Other business
+
+## Notes
+
+- Reviewed previous meeting minutes
+- Process for leaving the foundation
+  - Board has voted to approve the process to leave the Foundation
+  - Tom Pappas met with the lawyers to review the process and to review the agreements that we will use going forward.
```

Comment:
I think Tom also spoke to the lawyers about making the joining process clearer as well right

Reply:
That was what I was trying to say here. I'll clarify.

Reply:
Can we see the final approved process. What if a maintainer project doesn't agree? I still have major issues with the timescale and requirement to post intent for public comment

Reply:
https://github.com/dotnet-foundation/Home/discussions/68#discussioncomment-2168432

The board I understand voted this version unaltered. I don't think they'd be willing to change they just want an agreement that every project leaving has to abide by.

We discussed the public announcement during the maintainer's committee meeting. The feeling is due to previous incidents where they haven't been open about their decision making it was best to have a public announcement at least from projects leaving the foundation.

Reply:
So they're just bulldozing then with total disregard for the member project owners? I completely disagree with the requirement.

If a project wants to leave that is completely up to the discretion of the project maintainers and not a public concern. Leaving a project should absolutely not be delayed because the DNF has made obvious, painful mistakes in the past.

Reply:
And before anyone says.... "You should have been part of the working group". No. I do not want to dedicate my precious time to an organization that has caused me nothing but pain.